proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -411,8 +413,26 @@ message ListConnectionsResponse {
 
 // Request parameters for the RPC `ReceiveComponentDataStream`.
 message ReceiveComponentDataStreamRequest {
+  // A message for specifying a filter to apply to the stream.
+  message ComponentDataStreamFilter {
+    // List of metrics to return. Only the specified metrics will be returned.
+    // A complete list of metrics can be found [here](https://github.com/frequenz-floss/frequenz-api-common/blob/v0.5.0/proto/frequenz/api/common/v1/metrics/metric_sample.proto#L65-L66).
```

Comment:
I think it would be good to mention that not all metrics are supported by all components.

Reply:
Same here, I suggest to remove the hyperlink. Feels too much. See down below.